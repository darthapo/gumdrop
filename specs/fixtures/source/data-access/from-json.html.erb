<h1>Movies</h1>
<ul>
<% data.json_data.movies.each do |movie| %>
  <li><%= movie.title %></li>
<% end %>
</ul>