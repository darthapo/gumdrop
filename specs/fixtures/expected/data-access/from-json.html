<body class="site"><h1>Movies</h1>
<ul>
  <li>Super 8</li>
  <li>Star Trek</li>
  <li>The Avengers</li>
</ul></body>